internal/runtime/hcsv2/uvm.go

```diff
@@ -204,8 +205,10 @@ func newInvalidRequestTypeError(rt prot.ModifyRequestType) error {
 func modifyMappedVirtualDisk(ctx context.Context, rt prot.ModifyRequestType, mvd *prot.MappedVirtualDiskV2) (err error) {
 	switch rt {
 	case prot.MreqtAdd:
+		mountCtx, cancel := context.WithTimeout(ctx, time.Second*4)
```

Comment:
FYI, looks like this timeout is changing from 2 to 4 seconds. Not sure if that's intentional.

Reply:
There were two calls that each had 2 second timeouts. Now the entire set (both calls) have up to the total 4 seconds.